### PR TITLE
feat: add appeal documents section

### PR DESCRIPTION
**Pull request by Si3rr4wow**
## Ticket Number

https://pins-ds.atlassian.net/browse/AAPD-

## Checklist

- [ ] Requires infrastructure changes
- [ ] I have updated the documentation accordingly
- [ ] My commit history in this PR is linear
- [ ] New features have tests
- [ ] Breaking change (team conversation required)

## Important

Please do not merge from `main` (please only [rebase](https://github.com/Planning-Inspectorate/appeal-planning-decision/wiki/An-intro-to-Git-Rebase)). This keeps the history linear and easier to debug.

### CODE REVIEW

**Comment by Si3rr4wow** [comment]
I forgor the migration

File: packages/database/src/schema.prisma [hunk 1]
```diff
@@ -520,9 +521,9 @@ model AppellantSubmission {
   uploadOriginalApplicationForm     Boolean?
   uploadApplicationDecisionLetter   Boolean?
   uploadAppellantStatement          Boolean?
-  costApplication                   Boolean?
   uploadCostApplication             Boolean?
   uploadChangeOfDescriptionEvidence Boolean?
+  appellantStatement                Boolean?
```
Comment (topickering) [inline packages/database/src/schema.prisma:526]: Is this needed given we already have uploadAppellantStatement?

Comment (Si3rr4wow) [reply packages/database/src/schema.prisma:526]: oh no sorry it's not, good catch

**Review by topickering — APPROVED** [review]
LGTM

**Pull request merged**